content/modules/mobile-travel-tutorial/partials/installation/manual.adoc

```diff
@@ -71,32 +71,65 @@ git clone -b master --depth 1 https://github.com/couchbaselabs/mobile-travel-sam
 
 == Couchbase Server
 
+In this lesson, you will install and launch Couchbase Server.
 
-In this lesson, you will install and launch version 7.0.0 of Couchbase Server.
-If there is a later version available, you can download it as well bearing in mind that the instructions have been validated with the 7.0.0 version.
+If there is a later version available, you can download it as well bearing in mind that the instructions have been validated with a certain version.
```

Comment:
I think we should be specific what version that has been validated against. 
To keep instructions future proof, I would suggest something like
In this lesson, you will install and launch v7.0.x of Couchbase Server.
If there is a later version available, you can download it as well bearing in mind that the instructions have been validated with the version that is specified in instructions below.

Reply:
In this lesson, you will install and launch v7.0.x of Couchbase Server.

*M1 Users:* Install and launch v7.1.1 or later

If there is a later version available, you can download it as well bearing in mind that the instructions have been validated with the version that is specified in instructions below.